await-signals/await_signals_workflow.go

```diff
@@ -0,0 +1,164 @@
+package await_signals
+
+import (
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/server/common"
```

Comment:
We shouldn't have a dependency on the server repo in this repo at all IMO. We only did for samples that require access to the CLI plugin interface (happened before CLI moved out of server) and in a couple of cases it was used for its utilities (unnecessarily IMO).

I opened https://github.com/temporalio/samples-go/issues/237 for this. Don't have to fix now.

Reply:
oops, missed that it is not built-in.